### DISCUSSION TITLE
扫码登录一直显示失败

### DISCUSSION THREAD

**Discussion by Cody-Travers** [Richasy/Bili.Copilot#159] [Q&A]
扫码登录的时候一直显示登录失败请稍后重试
![屏幕截图 2023-10-29 102810](https://github.com/Richasy/Bili.Copilot/assets/139093182/fed94a53-28f0-4772-9476-cc217a89604e)

**Comment by Richasy** [comment]
已修复，下载最新版试试

**Comment by Cody-Travers** [comment]
问题已修复，感谢大佬！！












Nunki
***@***.***






---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2023年10月29日(周日) 下午2:06
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [Richasy/Bili.Copilot] 扫码登录一直显示失败 (Discussion #159)




 
已修复，下载最新版试试
